### PR TITLE
Multiple Storage of Same Code Removed

### PR DESCRIPTION
**Pull request by flaredragon**
In Reference to #126 
It checks from the open URL and compares the codes
If they are same it won't save it again.
Otherwise, it gets saved at a new place.
Please Review!

### CODE REVIEW

File: src/store/index.js [hunk 1]
```diff
@@ -1,195 +1,207 @@
...
+      }
+
+      axios
+        .get(`https://ide.cb.lk/code/${pasteId}`)
+        .then(({data}) => {
+          commit('changeLanguage', data.language)
+          commit('satCode', data.code)
+          commit('changeCustomInput', data.customInput)
+          commit('fileNameChange', data.fileName)
+            checkdata=data.code;
```
Comment (abhishek97) [inline src/store/index.js]: Wrap this inside a mutation, so that you do a `commit('setCheckData`, data.code)` here

File: src/store/index.js [hunk 2]
```diff
@@ -1,195 +1,207 @@
...
+ */
+'use strict'
+
+import Vue from 'vue'
+import Vuex from 'vuex'
+import axios from 'axios'
+import base64 from 'base-64'
+import VuexPersistence from 'vuex-persist'
+import samples from '../assets/js/sample-source'
+var checkdata;
```
Comment (abhishek97) [inline src/store/index.js]: make `checkData` a part of store state like all other vars , like `code`, `language` etc..

**Comment by flaredragon** [comment]
Please Review Again!
SHA256 is now being used for comparison.
checkData is now added as per Vue standards(under state)

File: src/store/index.js [hunk 3]
```diff
@@ -1,195 +1,218 @@
...
+    showInOutBox: false,
+    showSettings: false,
+    customInput: '',
+    customInputBuf: '', //input buffer to store customInput when toggled OFF
+    output: '',
+    fileName: '',
+    isChanged: false,
+    autoSave: true,
+    autoSaveIntervalId: null,
+      checkData:'',
```
Comment (abhishek97) [inline src/store/index.js]: Please avoid extra whitespaces and adhere to the same format as the rest of the code.

File: src/store/index.js [hunk 4]
```diff
@@ -1,195 +1,218 @@
...
+      state.theme = val
+    },
+    changeFont(state, val) {
+      state.font = val
+    },
+    changeFontSize(state, val) {
+      state.fontSize = val
+    },
+    setCheckData(state,val)
+      {
```
Comment (abhishek97) [inline src/store/index.js]: fix the code format

File: src/store/index.js [hunk 5]
```diff
@@ -1,195 +1,218 @@
...
+          commit('changeCustomInput', data.customInput)
+          commit('fileNameChange', data.fileName)
+          commit('setCheckData',data.code)
+      })
+    },
+
+    saveDataToServer({state, commit, dispatch}) {
+        const pasteId = state.route.params.id
+       
+        //console.log(x)
```
Comment (abhishek97) [inline src/store/index.js]: remove extra commented code; use comments only to document code

**Comment by flaredragon** [comment]
Good To go now?(Today Everyone Pestered me with spacing issues damn xD)

**Pull request merged**